test/jit/test_freezing.py

```diff
@@ -1762,3 +1762,103 @@ def test_conv_to_mkldnn_no_mkldnn(self):
             self.run_pass("convert_frozen_ops_to_mkldnn", frozen.graph)
             inp = torch.rand([4, 3, 4, 4])
             self.assertEqual(frozen(inp), mod(inp))
+
+@unittest.skipIf(not torch._C.has_mkldnn, "MKL-DNN build is disabled")
```

Comment:
* do we have a test that replaces `x.relu_().relu_().relu_()` like multiple propagations? I quickly scanned through tests and didn't find one.
* do we want a test that already has an in-place op. May not be necessary since we functionalize first?

Reply:
sure i can add for the first case. there are already tests for the second case and correct it's not necessary since we functionalize